--- a/src/nvim/version.c
+++ b/src/nvim/version.c
@@ -4245,6 +4245,9 @@ void intro_message(bool colon)
     "",
     N_(NVIM_VERSION_LONG),
     "────────────────────────────────────────────",
+    N_("This is unofficial Flatpak build of Neovide"),
+    N_("type  :help flatpak<Enter>  for flatpak info"),
+    "────────────────────────────────────────────",
     N_("Nvim is open source and freely distributable"),
     "https://neovim.io/#chat",
     "────────────────────────────────────────────",
